src/app/views/query-runner/request/Request.tsx

```diff
@@ -86,18 +88,29 @@ export class Request extends Component<IRequestComponent, any> {
     if (!item) { return; }
     const tabTitle = item.props.title;
     if (tabTitle) {
-      telemetry.trackEvent(TAB_CLICK_EVENT, { ComponentName: `${tabTitle} tab`, QuerySignature: '' });
+      this.trackTabClickEvent(tabTitle);
     }
   }
 
+  private trackTabClickEvent(tabTitle: string) {
+    const { sampleQuery }: any = this.props;
+    const sanitizedUrl = sanitizeQueryUrl(sampleQuery.sampleUrl);
```

Comment:
The sampleQuery is not part of the properties in the component. This will lead to a crash when a tab is clicked